src/core/viz/functions.js

```diff
@@ -331,7 +331,7 @@ export { time as date };
 export const top = (...args) => new Top(...args);
 
 export const fade = (...args) => new Fade(...args);
-export const torque = (...args) => new Torque(...args);
+export const animation = (...args) => new Animation(...args);
```

Comment:
Let's keep an alias to `torque` to avoid the breaking change

Reply:
I thought beta status was to forget this kind issues :trollface:

Reply:
(just trolling, IM OK with the alias, maybe with a warning? )

Reply:
A warning is a good idea, yes

Reply:
added 👌